### ISSUE TITLE
Tabs Component keyboard accessibility (AVT 2)

### ISSUE THREAD

**Issue by johnharan** [IBM/carbon-components-angular#254]
Currently the Up/down arrows change the focused tab, only the left/right arrows should change the focused tab.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

Tabs component

> What did you expect to happen? What happened instead? What would you like to see changed?

Up/down arrows should not be binded to any Tabs action

> What browser are you working in?

FF 63

> What version of the Carbon Design System are you using?

"carbon-components": "9.32.1",
"carbon-components-angular": "1.9.1"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

## Steps to reproduce the issue

1. Go to Tabs component [example](https://angular.carbondesignsystem.com/?knob-size=md&knob-showSelectionColumn=true&knob-striped=true&knob-sortable=true&selectedKind=Tabs&selectedStory=Basic&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)
2. Tab into the tab component.
3. Press up or down arrow. Notice that the focused tab changes.

**Comment by zvonimirfras** [comment]
Active tab doesn't change, focus changes. Users still have to activate the focused tab using the enter key.

That said, keyboard interactions are still incorrect.

Reference for whoever takes on this issue: https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

**Comment by cal-smith** [comment]
from #270

 We’ll want to follow https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus

So a default of follow focus with an option to go to manual activation for async tabs.
